src/ansys/geometry/core/designer/component.py

```diff
@@ -588,6 +590,53 @@ def sweep_chain(
         self._master_component.part.bodies.append(tb)
         return Body(response.id, response.name, self, tb)
 
+    @protect_grpc
+    @check_input_types
+    @ensure_design_is_active
+    def sweep_with_guide(
+        self, name: str, profile: List[Curve], guide: Curve, path: Curve, tight_tolerance: bool
+    ) -> Body:
+        """
+        Create a body by sweeping a chain of curves along a path.
+
+        Notes
+        -----
+        The newly created body is placed under this component within the design assembly.
+
+        Parameters
+        ----------
+        name : str
+            User-defined label for the new solid body.
+        path : List[TrimmedCurve]
+            The path to sweep the chain along.
+        chain : List[TrimmedCurve]
+            A chain of trimmed curves.
```

Comment:
```suggestion
            Path to sweep the chain along.
        chain : List[TrimmedCurve]
            Chain of trimmed curves.
```
Omit article (The, A, An) in brief description.

Reply:
@PipKat sorry it is draft but not meant to be reviewed yet. I am taking it down.